index.html

```diff
@@ -468,6 +468,21 @@ <h3>RANDOM</h3></a>
           
          </div>
       </div>
+      <!-- Fact 18 -->
+    <div class="wrapper">
+        <div class="card-bg">
+          <div class="card">
+             <div class="card-img">
+            <img calss="headshot" src="images/favicon.png">
+          </div> 
+            <div class="card-inner">
+              <p>"Smartphones run on more code than NASA’s computers in 1969."</p>
+            <span> Lakshay Roopchandani - <a href="#"> https://github.com/lakshayroop5 <a></span>

```

Comment:
Add your link in href attribute remove # and add your link and where you added link this is place for your username of github